dhis2/dhis2-core
# feat: clear GeoJSON data [DHIS2-13313]

### Summary
Adds GeoJSON API endpoint to bulk delete all GeoJson data for a particular `GEOJSON` attribute or the `geometry` property of the `OrganisationUnit`.

Both clearing the `geometry` and removing the attribute is done using a single DB update query.
The API response is still a report to be consistent with the import or single delete response.

### Examples
Delete the `geometry` property of **all** organisation units:

    DELETE /api/organisationUnits/geometry

Delete a particular `GEOJSON` attribute for **all** organisation units:

    DELETE /api/organisationUnits/geometry?attributeId={uid}

### Automatic Testing
No tests were added as the feature is implemented in a way that depends on actual database behaviour which is best tested manually.

### Manual Testing
* eventually create a new `GEOJSON` attribute to use
* run GeoJson import to set some GeoJSON data. (ask me for test-file) or set a value manually
* run the delete 
* check that the deleted property or attribute value is cleared

### Documentation
As part of https://github.com/dhis2/dhis2-docs/pull/1006
